### PR TITLE
#184: Changed Label to Parcel on shipping label

### PR DESCRIPTION
**Pull request by kaydentsoftwire**
## What's changed
The shipping label shows Parcel instead of Label

## Screenshots / Videos
| Before     | After      |
|------------|------------|
| 327413663-9436a69c-ccd1-4da8-91f2-71530b1399c1 | image |

## Checklist
- [x ] The ticket is up-to-date - Please document any deviations from the original approach if there is any.
- [x ] I have documented the testing steps for QA
- [x ] I have self-reviewed this PR
- [x] Make sure you've verified it works via `npm run dev`
- [x ] Make sure you've verified it works via `npm run build` and `npm run start`
- [x ] Make sure you've fixed all linting problems with `npm run lint_fix`
- [x ] Make sure you've tested via `npm run test:e2e`

If you have made any changes to the database...
  - [ ] The migration files are up-to-date with my final set up (`npx supabase db diff -f <name_of_migration>` should create nothing at this point)
  - [ ] I have updated the typescript definitions for the database with `db:local:generate_types`
  - [ ] I have modified the seed in `supabase/seed/seed.mts` if appropriate
  - [ ] If I have modified the seed, I have also generated the seed with `npm run db:generate_seed` 
  - [ ] With my final set up, I can run `npm run dev:reset_supabase` without any errors.
  - Does this require resetting the deployed database? - YES / NO (remove as appropriate)
  - [ ] I have checked that migration can happen successfully without resetting the database, doing the following.
    - Make sure you have rebased your branch onto dev or merged dev into your branch.
    - Checkout the dev branch
    - Run `npm run post_checkout` to reset the database, including the seed data.
    - Checkout your branch
    - Run `npx supabase migrations up --include-all --local` to run all outstanding migration files
    - Check that the resulting database is what you expect, bar any seed data changes.

### CODE REVIEW

File: package.json
Comment (AmeliaIsCoding) [inline package.json:32]: Can you revert the changes in this file please?:)

**Review by AmeliaIsCoding — CHANGES_REQUESTED** [review]
Looks good, just revert your changes in package.json :)

**Review by AmeliaIsCoding — APPROVED** [review]
LGTM, you can merge once @minalui approves too!

**Review by minalui — APPROVED** [review]
LGTM! 🙂

File: package.json
Comment (kaydentsoftwire) [reply package.json]: Done!

**Pull request merged**
